photonpump/messages.py

```diff
@@ -380,6 +396,7 @@ def NewEvent(
 
 
 class SubscriptionCreatedResponse:
-    def __init__(self, result:SubscriptionResult, reason: str):
+
+    def __init__(self, result: SubscriptionResult, reason: str):
```

Comment:
`-> None`

Reply:
Actually, I'm going to ignore type-signatures 'cos there's an open issue #38 for 0.6

I started a branch here https://github.com/madedotcom/photon-pump/tree/dev/strict-typing to add strict type checking to messages.py